docs/frontend/frontend-npm-webpack.md

```diff
@@ -0,0 +1,116 @@
+# Modern frontend
```

Comment:
According to our [guidelines for writing documentation](https://docs.shopsys.com/en/9.0/contributing/guidelines-for-writing-documentation/)

> All documentation files should be named same as first heading (in lowercase, non-alphanumeric characters replaced by dash).